csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
Are you happy so long as it can be easily *consumed* by those platforms, or do you think it has to be *buildable* on them as well? The latter is likely to be more of an issue.